assignments/objects.js

```diff
@@ -19,27 +19,80 @@ const example = {
 
 // Write your intern objects here:
 
+const employeeOne = {
```

Comment:
Great job on creating objects, and the naming convention really help distinguish between each object.